index.js

```diff
@@ -104,12 +110,25 @@ function parseMembers(options) {
 
       if (customBaseIri && split[1].startsWith(customBaseIri.prefix + ':')) {
         latestCustomVocabElementId = split[1];
-        customVocab.push({
-          '@id': latestCustomVocabElementId,
-          '@type': 'Class',
-          'subClassOf': {'@id': 'schema:Thing'}
-        });
+          customVocab.push({
+            '@id': latestCustomVocabElementId,
+            '@type': 'Class',
+            'subClassOf': {'@id': 'owl:Thing'}
```

Comment:
Why is `schema:Thing` removed?

Reply:
I just thought owl:Thing was more generic than schema:Thing.

Reply:
Both are generic. For it's fine to add `owl:Thing`, but I would not necessarily remove `schema:Thing`. If people only want `owl:Thing` they can use `@superTypes`.

Reply:
Fair enough!